website/docs/en/config/entry.mdx

```diff
@@ -106,23 +86,15 @@ When you set `entry` to an object, you can set the value of the entry to a descr
 
 #### EntryDescription.import
 
-- **Type:**
+- **Type:** `string | string[]`
```

Comment:
I prefer using the `ts` code block, as it can tell the type name and maybe helpful for TypeScript users.

Reply:
Most of the documents I see are written directly inline code, so for simple types, I use Inline code.

Reply:
Yeah, simple types can just use inline code. But we don't need to change the existed code block back to inline code, which reduces the potentially useful information.